_config.yml

```diff
@@ -6,7 +6,7 @@ description: [Yasson is a Java framework which provides a standard binding layer
 links:
   source: https://github.com/eclipse-ee4j/yasson
   #docs: documentation
-  javadocs: docs/api/?overview-summary.html
+  javadocs: https://www.javadoc.io/doc/org.eclipse/yasson/1.0.5
```

Comment:
Similar to the previous comment, lets just link this to `https://www.javadoc.io/doc/org.eclipse/yasson/` (which will automatically redirect to the latest version) so we don't need to remember to manually update this each release

Reply:
(sorry, I realize I asked you to use the 1.0.5 link earlier but just now realized we could use the URL without the version in it)

Reply:
No worries, i will change that :-)